README.md

```diff
@@ -106,6 +106,24 @@ require 'avroluation/rake/register_schemas_task'
 Avrolution::Rake::RegisterSchemasTask.define
 ```
 
+### Avro Register All Schemas Rake Task
+
+This rake task allows you to register all schemas discovered under `Avrolution.root`.
+
+Similarly to the task `avro:register_schemas`, it will register them against the configured
+registry. Additionally, this task will be auto included for Rails applications.
+
+```bash
+rake avro:register_all_schemas
```

Comment:
What's the rationale for a new rake task vs. making the `schemas` argument optional for the existing `avro:register_schemas` task?

Reply:
My only thought around not modifying the existing task is to prevent a scenario where someone leaves `schemas=foo,bar` off, and we just start registering everything when they might not have expected/wanted that. Thoughts? I am happy to modify the existing, and maybe add an `all=true` flag or something to alleviate that concern?

Reply:
I don't have strong feelings one way or another. If this was a real CLI, then I'd say it should be an optional argument to a single register command but `rake` passing named arguments as environment variables makes argument validation tough leading to unexpected behavior if you mistype an arg name.

Reply:
Yeah I am not a fan of `rake`s argument "handling" :)